pset-functions.js

```diff
@@ -175,6 +246,19 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+function isPalindrome(word){
+  let splitWords = word.split('')
+  let reversedArr = splitWords.reverse()
+  let reversedWord = reversedArr.join("")
+  if(word === reversedWord){
+    return true
+  } else{
+    return false
+  }
+}
+
+console.log(isPalindrome("noon"))
```

Comment:
@nmikhol Good work her on this solution 
- taking the string and splitting into an array
- reversing the string and joining to convert back to a string
- and checking against the original word to return true if it's a palindrome and false if it's not.